lib/read-files.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
```

Comment:
Great job on making your readFiles function readable and spaced out nicely with all the nested callbacks. :)